lib/npm.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash';
+import path from 'path';
+import { exec } from 'teen_process';
+
+const APPIUM_ROOT = path.resolve(__dirname, '..', '..');
+
+async function npmExec (cmd, args, {json} = {json: false}, execOpts = {}) {
+  args.unshift(cmd);
+  if (json) {
+    args.push('-json');
+  }
+  execOpts.cwd = APPIUM_ROOT;
+  const {stdout, stderr, code} = await exec('npm', args, execOpts);
```

Comment:
npm is a batch script on windows, so this won't work there

Reply:
i'm able to run commands on my windows machine just fine using `npm`